modules/platforms/cpp/thin-client/src/impl/message.h

```diff
@@ -569,6 +619,110 @@ namespace ignite
                 const Writable& val3;
             };
 
+            /**
+             * Tx start request.
+             */
+            template<int32_t OpCode>
```

Comment:
Why does this type needs to be parameterized if the parameter is always the same?